Build fails with gcc 4.8

Building on an Ubuntu 14.04 machine gives the following error:

```
ERROR: /usr/local/google/home/rodrigoq/.cache/bazel/_bazel_rodrigoq/f18209624566089c9f60bd21b92d9983/external/org_lzma_lzma/BUILD.bazel:21:1: C++ compilation of rule '@org_lzma_lzma//:lzma' failed (Exit 1)
In file included from external/org_lzma_lzma/src/liblzma/common/common.h:17:0,
                 from external/org_lzma_lzma/src/liblzma/common/vli_decoder.c:13:
external/org_lzma_lzma/src/common/mythread.h:138:52: error: expected ';', ',' or ')' before 'set'
 mythread_sigmask(int how, const sigset_t *restrict set,
                                                    ^
In file included from external/org_lzma_lzma/src/liblzma/common/vli_decoder.c:13:0:
external/org_lzma_lzma/src/liblzma/common/common.h:107:27: error: expected ';', ',' or ')' before 'in'
   const uint8_t *restrict in, size_t *restrict in_pos,
                           ^
external/org_lzma_lzma/src/liblzma/common/common.h:150:2: error: unknown type name 'lzma_code_function'
  lzma_code_function code;
  ^
external/org_lzma_lzma/src/liblzma/common/common.h:268:51: error: expected ';', ',' or ')' before 'in'
 extern size_t lzma_bufcpy(const uint8_t *restrict in, size_t *restrict in_pos,
                                                   ^
external/org_lzma_lzma/src/liblzma/common/vli_decoder.c:17:36: error: expected ';', ',' or ')' before 'vli'
 lzma_vli_decode(lzma_vli *restrict vli, size_t *vli_pos,
                                    ^
Target @org_lzma_lzma//:lzma failed to build
```

This can be resolved by using `bazel build --conlyopt=-std=c99`. It can also be resolved by setting `copts = ["-std=c99"]` in BUILD.lzma, but I'm concerned this could break other platforms?

--- a/BUILD.lzma
+++ b/BUILD.lzma
@@ -96,6 +96,7 @@ cc_library(
         "src/liblzma/simple/x86.c",
     ],
     hdrs = glob(["src/**/*.h"]) + ["config.h"],
+    copts = ["-std=c99"],
     defines = [
         "HAVE_CONFIG_H",
     ],

